core/src/apps/tron/layout.py

```
@@ -234,6 +236,33 @@ def require_confirm_undelegate(
+    signer: str,
+) -> Awaitable[None]:
+    from trezor.ui.layouts.lvgl import confirm_tron_unfreeze
+
+    return confirm_tron_unfreeze(ctx, "Cancel All UnStaking", signer, None, None)
+
+
+def require_confirm_vote_witness(
+    ctx: Context,
+    signer: str,
+    votes: list[tuple[str, int]],
+    support: bool | None,
```

Ruff (0.14.5)

Boolean-typed positional argument in function definition

(FBT001)